Source/Extensions/Blazorise.RichTextEdit/RichTextEdit.razor.css

```diff
@@ -0,0 +1 @@
+﻿::deep .ql-tooltip { z-index: 1 }
```

Comment:
Can you explain what is the `::deep` selector? All I can found is that it is related to vuejs.

Reply:
Ahh I noticed now you used scoped component styles(`RichTextEdit.razor.css`)